packages/utils/telemetry-utils/src/staging.ts

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ILoggingError } from "@fluidframework/common-definitions";
+
+// ///////////////////////////////////////////////////////////////////////// //
+//        THIS CODE TO BE MOVED TO COMMON-DEFINITIONS AND COMMON-UTILS       //
+// ///////////////////////////////////////////////////////////////////////// //
+
+/**
+ * All normalized errors flowing through the Fluid Framework adhere to this readonly interface.
+ * It includes Error's properties but as optional, plus errorType and fluidErrorCode strings.
+ */
+export interface IFluidErrorBase extends Readonly<Partial<Error>> {
+    readonly errorType: string;
+    readonly fluidErrorCode: string;
+    readonly message: string;
+}
+
+/** type guard for IFluidErrorBase interface */
+export function isFluidError(e: any): e is IFluidErrorBase {
+    return typeof e?.errorType === "string" &&
+        typeof e?.fluidErrorCode === "string" &&
+        typeof e?.message === "string";
+}
+
+/** type guard for old standard of valid/known errors */
+export function isValidLegacyError(e: any): e is Omit<IFluidErrorBase, "fluidErrorCode"> {
```

Comment:
I believe your intention is for IFluidErrorBase to not satisfy this check, but it will.

Reply:
No it's fine to